testsuite/MDAnalysisTests/topology/test_guessers.py

```diff
@@ -131,9 +131,19 @@ def test_guess_impropers():
     assert_equal(len(vals), 12)
 
 
+def bond_sort(arr):
```

Comment:
@richardjgowers  Why was this needed again? nsgrid stuff?

Reply:
Yeah so `self_capped_distance` used to always sort bond indices as a side effect of the algorithm, it no longer does.  So guessed bonds are now in an arbitrary order.